src/Illuminate/View/View.php

```diff
@@ -58,7 +58,7 @@ class View implements ArrayAccess, ViewContract
      * @param  \Illuminate\View\Engines\EngineInterface  $engine
      * @param  string  $view
      * @param  string  $path
-     * @param  array   $data
+     * @param  mixed   $data
```

Comment:
> https://github.com/laravel/framework/blob/5.2/src/Illuminate/Support/Collection.php#L1135-L1156

That link is not idea. We always prefer the most useful phpdoc. Saying mixed is a bit of a waste if a phpdoc.